.env.template

```diff
@@ -1 +1,9 @@
+PORT=8080
+
 DATABASE_URL="postgres://postgres:permanent@localhost:5432/test_permanent"
+
+FUSIONAUTH_HOST=<host URL here
+FUSIONAUTH_API_KEY=<API key here>
```

Comment:
Despite the name, the back-end key isn't actually specific to the back-end application; API keys appear to be scoped to tenants, so I don't see a reason to create a separate admin key (since stela would need both keys anyway).